### PR TITLE
fix(positionning): fix start/end placements for popper

### PR DESCRIPTION
**Pull request by fbasso**
The purpose is to have the right placement for widgets using popper when the dir="rtl" is set on the html tag

### CODE REVIEW

**Review by maxokorokov — COMMENTED** [review]
I left 2 comments, could you please rebase and I'll review again :)

File: src/test/ngb-rtl.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export class NgbTestLTR {
+  constructor(@Inject(DOCUMENT) private _document: any) {}
+
+  isRTL() { return false; }
+
+  getLang() { return 'en'; }
+}
+
+@Injectable({providedIn: 'root'})
+export class NgbTestRTL {
```
Comment (maxokorokov) [inline src/test/ngb-rtl.ts:14]: This is used nowhere

File: src/test/ngb-rtl.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import {Injectable, Inject} from '@angular/core';
+import {DOCUMENT} from '@angular/common';
+
+@Injectable({providedIn: 'root'})
+export class NgbTestLTR {
```
Comment (maxokorokov) [inline src/test/ngb-rtl.ts:5]: If you remove all `provides` from tests, this will not be used anywhere either

**Review by maxokorokov — CHANGES_REQUESTED** [review]
Hey,

left another review, please see inline comments.

Big things:
1. you shouldn't provide anything special in any of the tests, so `NgbTestRTL` and `NgbTestLTR` are not needed
2. integration tests are missing, meaning we don't have tests that say that "if we have a correct RTL attribute on <html> element, placement classes are correctly generated". Well, at least one test :)
3. We don't need `getLang()` at the moment, so let's remove it

Cheers,
Max

**Comment by fbasso** [comment]
Thanks for your review @maxokorokov, I'm going to see the changes to be done.

**Pull request merged**